### ISSUE TITLE
when use GroundingDINO as the detector

### ISSUE THREAD

**Issue by lastwhisper-ghy** [SysCV/sam-hq#93]
Hi . I want to know when you use the box from the GroundingDINO as the prompt, if only boxes greater than the threshold are used as input？

**Comment by lkeab** [comment]
hi, please check out evaluation script [here](https://github.com/SysCV/sam-hq/blob/main/seginw/test_seginw_hq.sh)

**Comment by lkeab** [comment]
yes, when visualization we only show the bboxes higher than the threshold; when computing AP for quantitative evaluation, we take the top100.
